src/messages/MOSDOp.h

```diff
@@ -242,14 +236,14 @@ struct ceph_osd_request_head {
       ::encode_nohead(snaps, payload);
     } else {
       header.version = HEAD_VERSION;
-      ::encode(client_inc, payload);
+      ::encode(pgid, payload);
```

Comment:
NP, thank you! I'm looking forward to seeing these pieces start moving in. :)

I should also mention that for performance-based patches we also like to have a link to some data demonstrating that the patch actually improves things; that's helpful to avoid accidentally making things slower. ;)